resources/compile.sh

```diff
@@ -135,7 +135,7 @@ function build_release_model {
 
   pushd build
   mkdir obj
-  ../../../../node_modules/.bin/tsc --outDir ./obj ../source/model.ts 
+  ../../../../node_modules/.bin/tsc --module commonjs --target es6 --outDir ./obj ../source/model.ts 
```

Comment:
1. would it be cleaner to use `npx tsc` instead of `../../../../node_modules/.bin/tsc`>
2. Is it worth extracting the compiler options to an explicit `tsconfig.json`?

Reply:
1. Updated to use `npx`. Note, on Windows, this requires Node.js version 10.0 or later.

2. Am I right in thinking this would require an explicit `tsconfig.json` for each model? If so for now I'll leave it in the script.